web/src/features/map/Map.tsx

```diff
@@ -214,6 +234,11 @@ export default function MapPage(): ReactElement {
         onError={onError}
         onMouseMove={onMouseMove}
         onMouseOut={onMouseOut}
+        onDragStart={onDragStart}
+        onZoomStart={onZoomStart}
+        onZoomEnd={onZoomEnd}
+        dragPan={{ maxSpeed: 0 }} // Disables easing effect to improve performance on exchange layer
+        onDragEnd={onDragEnd}
```

Comment:
Follow up suggestion to the first one:
```suggestion
        onDragStart={onDragOrZoomStart}
        onZoomStart={onDragOrZoomStart}
        onZoomEnd={onDragOrZoomEnd}
        dragPan={{ maxSpeed: 0 }} // Disables easing effect to improve performance on exchange layer
        onDragEnd={onDragOrZoomEnd}
```